server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/StdJobScheduler.java

```diff
@@ -172,27 +172,30 @@ private void initCronJob(String key, String cronExpression, Class<? extends Job>
         try {
             String group = JobConstants.ODC_JOB_MONITORING;
             TriggerKey triggerKey = TriggerKey.triggerKey(key, group);
-            JobKey jobKey = JobKey.jobKey(key, group);
+
             Trigger trigger = TriggerBuilder.build(triggerKey, config);
```

Comment:
Using Quartz to implement core scheduling logic of task framework cannot achieve high availability.

Reply:
note a todo for test high availability